packages/safe-tools-client/app/components/future-payments-list/scheduled-payment-card/index.css

```diff
@@ -0,0 +1,55 @@
+.scheduled-payment-card {
+  display: block;
+
+  padding: var(--boxel-sp-sm) var(--boxel-sp);
+  margin-bottom: var(--boxel-sp-sm);
+}
+
+.scheduled-payment-card .boxel-icon-button {
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.scheduled-payment-card__content {
+  display: flex;
+  flex-direction: column;
+}
+
+.scheduled-payment-card_content-payment-detail {
+  margin-top: var(--boxel-sp-lg);
+
+  display: flex;
+}
+
+.scheduled-payment-card_content-token-symbol {
+  margin-right: var(--boxel-sp-xs);
+  vertical-align: bottom;
+  width: var(--boxel-icon-sm);
+  height: var(--boxel-icon-sm);
+}
+
+.scheduled-payment-card__content-pay-at {
+  font: 700 var(--boxel-font-sm);
+  letter-spacing: var(--boxel-lsp-xl);
+}
+
+.scheduled-payment-card__content-payee {
```

Comment:
Can we clean this up to get rid of the "content" word? 

Ideally:

Block: `scheduled-payment-card`
Elements: `scheduled-payment-card__pay-at`, `scheduled-payment-card__token-amounts`,...